test/README.md

```diff
@@ -0,0 +1,27 @@
+# Amazon ECS Agent - Functional Tests
```

Comment:
I like this approach for functional tests - provides a pretty good test suite for our use cases.